pkg/client/didexchange/client.go

```diff
@@ -84,20 +84,20 @@ func New(ctx provider) (*Client, error) {
 
 // CreateInvitation create invitation
 func (c *Client) CreateInvitation(label string) (*didexchange.Invitation, error) {
-	verKey, err := c.wallet.CreateKey()
+	verKey, err := c.wallet.CreateEncryptionKey()
 	if err != nil {
 		return nil, fmt.Errorf("failed CreateSigningKey: %w", err)
 	}
 
 	invitation := &didexchange.Invitation{
 		ID:              uuid.New().String(),
 		Label:           label,
-		RecipientKeys:   []string{verKey},
+		RecipientKeys:   []string{string(verKey)},
```

Comment:
This does look ... odd :).

Reply:
It is.

Reply:
Ok so - this will need changing as we push more crypto refactoring, yes.

Reply:
We now use strings as identifiers instead of a wrapping type, this is fixed.